lib/katello/tasks/pulp3_migration.rake

```diff
@@ -4,6 +4,23 @@ namespace :katello do
     services = [:candlepin, :foreman_tasks, :pulp3, :pulp, :pulp_auth]
     Katello::Ping.ping!(services: services)
 
+    #check for valid yaml
+    puts "Checking for valid migration configuraton."
+    if SETTINGS[:katello][:use_pulp_2_for_content_type].nil?
+      msg = _("Migration aborted, 'use_pulp_2_for_content_type' is missing from Katello configuration.")
+      $stderr.print(msg)
+      fail ForemanTasks::TaskError, task
+    end
+
+    if !SETTINGS[:katello][:use_pulp_2_for_content_type][:docker] &&
+      !SETTINGS[:katello][:use_pulp_2_for_content_type][:file]
+      !SETTINGS[:katello][:use_pulp_2_for_content_type][:yum]
+      !SETTINGS[:katello][:use_pulp_2_for_content_type][:deb]
+      msg = _("Migration aborted, no content types in 'use_pulp_2_for_content_type' section are set to 'true'.")
+      $stderr.print(msg)
+      fail ForemanTasks::TaskError, task
+    end
```

Comment:
i had originally envisioned this to go in foreman-maintain, are you sure that foreman-maintain won't shut down pulp services aftewards in the 6.9 scenario ?